osivalidator/osi_general_validator.py

```diff
@@ -122,12 +121,35 @@ def command_line_arguments():
 VALIDATION_RULES = osi_rules.OSIRules()
 
 
+def detect_message_type(path: str):
+    """Automatically detect the message type from the file name.
+    If it cannot be detected, the function return SensorView as default.
+
+    Args:
+        path (str): Path incl. filename of the trace file
+
+    Returns:
+        Str: Message type as string, e.g. SensorData, SensorView etc.
+    """
+    filename = os.path.basename(path)
+    if filename.find("_sd_") != -1:
+        return "SensorData"
+    if filename.find("_sv_") != -1:
+        return "SensorView"
+    if filename.find("_gt_") != -1:
+        return "GroundTruth"
+    return "SensorView"
```

Comment:
Maybe a logger warning here?

Reply:
There was no logger warning before when no type is set. Of course we could still add one here, but I am actually not sure, if the logger even works correctly, see #53.

Since this is blocking my development in https://github.com/openMSL/sl-2-0-traffic-participant-model-repository-template/pull/12 I propose to merge without the logger entry and fix this later in #53.